test/unit/spec/MfaVerify_spec.js

```diff
@@ -3170,6 +3177,62 @@ function (Okta,
       });
     });
 
+    Expect.describe('Supports chains of MFA_REQUIRED responses', function () {
```

Comment:
nit: this should be something like ```MFA_REQUIRED to MFA_REQUIRED```
and the first ```itp```, something like ```renders a new view when we successfully answer the first MFA_REQUIRED response``` or so....
in the end, this will read the whole sentence together, so ```MFA_REQUIRED to MFA_REQUIRED renders a new view when we successfully answer the first MFA_REQUIRED response```

Reply:
Hey Mau, not all the tests in this group are MFA_REQUIRED to MFA_REQUIRED. The first tests the password reauth scenario, the second tests MFA_CHALLENGE to MFA_REQUIRED using the drop down and the third tests the inline totp scenario. It is basically testing the main scenarios wrt my changes.